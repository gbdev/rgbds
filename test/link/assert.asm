
SECTION "test", ROM0

FloatingBase:
	assert WARN, FloatingBase, "Worry about me, but not too much."
	assert FAIL, FloatingBase, "Okay, this is getting serious!"
	assert FATAL, FloatingBase, "It all ends now."
	assert FAIL, FloatingBase, "Not even time to roll credits!"
	assert WARN, 0, "Still can finish the film, though!"
