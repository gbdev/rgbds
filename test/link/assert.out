warning: assert.asm(5): Worry about me, but not too much.
error: assert.asm(6): Okay, this is getting serious!
FATAL: assert.asm(7): It all ends now.
Linking aborted after 2 errors
